src/js/components/Menu.js

```diff
@@ -114,6 +114,34 @@ var Menu = React.createClass({
     this.setState({active: false});
   },
 
+  _onUpKeyPress: function (event) {
+    if (!this._drop.activeChild) {
+      var children = this._drop.container.querySelector('.box').childNodes;
```

Comment:
what if there are more than one element with `.box` class? Can we assign an id to the box container? or something that is guaranteed to be unique?